src/tile.js

```diff
@@ -322,6 +330,27 @@ $.Tile.prototype = {
         $.setElementOpacity( this.element, this.opacity );
     },
 
+    /**
+     * The Image object for this tile.
+     * @member {Object} image
+     * @memberof OpenSeadragon.Tile#
+     * @return {Image}
+     */
+    get image() {
+        return this.cacheImageRecord.getImage();
+    },
+
+    /**
+     * The CanvasRenderingContext2D instance for tile image data drawn
+     * onto Canvas if enabled and available
+     * @member {CanvasRenderingContext2D} canvasContext
+     * @memberof OpenSeadragon.Tile#
+     * @return {CanvasRenderingContext2D}
+     */
+    get canvasContext() {
```

Comment:
Our API is pretty consistently based on functions that start with `get`, and we've only used getters in places where we need to in order to keep old APIs functional while they are deprecated. Because of that, let's not add a new getter where there was no property before. Instead, let's call this `getCanvasContext`. 

For that matter, we should probably add a `getImage` and the `image` getter can call it and also produce a deprecation warning.